pkg/crc/segment/segment_test.go

```diff
@@ -95,7 +98,8 @@ func TestClientUploadWithConsent(t *testing.T) {
 		require.Equal(t, s.Batch[0].Traits.OS, runtime.GOOS)
 		require.Equal(t, s.Batch[0].Traits.ExperimentalFeatures, true)
 		require.Equal(t, s.Batch[1].Type, "track")
-		require.Equal(t, s.Batch[1].Properties.Error, "an error occurred")
+		require.Equal(t, s.Batch[1].Properties.Error, crcErr.VMNotExist.Error())
+		require.Equal(t, s.Batch[1].Properties.ErrorType, fmt.Sprintf("%T", crcErr.VMNotExist))
```

Comment:
Might be worth hardcoding the actual string (`"github.com/code-ready/crc/pkg/errors.VMNotExist"` or whatever it is), as we don't want to inadvertantly change the value of "Properties.ErrorType" for `crcErr.VMNotExist`.